pages/api/zapbye.ts

```diff
@@ -0,0 +1,100 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import jsforce from 'jsforce';
+
+const CONSTANTS = {
+      SALESFORCE_EMAIL: process.env.SALESFORCE_EMAIL,
+      SALESFORCE_PASSWORD: process.env.SALESFORCE_PASSWORD,
+      SALESFORCE_SECURITY_TOKEN: process.env.SALESFORCE_SECURITY_TOKEN,
+      LEMLIST_API_KEY: process.env.LEMLIST_API_KEY,
+      SALESFORCE_TASK_SUBJECT: 'Lemlist Email',
+      SALESFORCE_LEMLISTTASK_ID: '0123k0000014TCyAAM',
+      SALESFORCE_STATUS: 'Completed',
+      SALESFORCE_TASK_SUBTYPE: 'Task',
+      SALESFORCE_SOBJECT_TYPE: 'Task',
+      LEMLIST_CAMPAIGN_FETCH_COOLDOWN_TIME_IN_MS: 30000,
+};
+
+let salesforceConnection;
```

Comment:
The reason I didn't set it as const is because the connection sometimes breaks and it has to be reinitialized. Sadly the JSForce library has no real way of checking status of connection of reconnecting, so it has to get re-init to reconnect.

Reply:
Sounds good then. I thought it will be pretty stable.

Reply:
Ok so this does not seem to be behaving the way I expect it to. salesforceConnection becomes undefined when the next request comes in, but I would like to reuse the connection as much as I can. Why is the global variable value getting thrown away?